style.css

```diff
@@ -0,0 +1,269 @@
+* {
+	box-sizing:	border-box;
+	font-family: "Arial";
+	font-size: 15px;
+	background-color:#000000;
```

Comment:
it's a little unusual to put a background colour on every element, in this case you've put a black background on every element. i would say it's preferable to leave the background alone for most elements (meaning it will be transparent) and only set the background on elements that actually have a coloured background (such as, in your case, the body element and the navbar element)